hybris/bin/custom/jhondeerestore/resources/jhondeerestore/import/sampledata/contentCatalogs/jhondeere-ukContentCatalog/cms-content.impex

```diff
@@ -1083,3 +1083,32 @@ INSERT_UPDATE ContentSlotForPage;$contentCV[unique=true];uid[unique=true];positi
 ;;LeftContentSlot-checkout-login;LeftContentSlot;;LeftContentSlot-checkout-login;;;
 ;;CenterContentSlot-checkout-login;CenterContentSlot;;CenterContentSlot-checkout-login;;;
 ;;RightContentSlot-checkout-login;RightContentSlot;;RightContentSlot-checkout-login;;;
+<!--Start of  Story ID- 105 -->
+		#Impex for Media
+
+INSERT_UPDATE Media;$contentCV[unique=true];code[unique=true];@media[translator=de.hybris.platform.impex.jalo.media.MediaDataTranslator];mime[default='image/png'];&imageRef;folder(qualifier)[default='images'];altText
+;;customMedia1;$siteResource/images/banners/homepage/custom.png;;custom.png;;
+
+#Impex for CustomComponent
+
+INSERT_UPDATE CustomComponent;$contentCV[unique=true];uid[unique=true];name;customImage(code);customText[lang=$lang];&componentRef;
```

Comment:
You are not supposed to name like this, Name this component according to your story, It is AirpointsComponent